contractsdk/cpp/table/basic_iterator.cc

```diff
@@ -0,0 +1,61 @@
+#include "table/basic_iterator.h"
+#include "xchain/xchain.h"
+
+namespace xchain { namespace cdt {
+
+Iterator::Iterator(xchain::Context* ctx, const std::string start, const std::string limit, size_t cap) {
+    _ctx = ctx;
+    _cap = cap;
+    _start = start;
+    _limit = limit;
+
+    if (!load()) {
+        _it = _buf.size();
+    } else {
+        _it = 0;
+    }
+}
```

Comment:
No mater what value is returned by load(), _it is all set to 0. So why did you write it by this way? What's your aim?